upf/upf.api

```diff
@@ -324,3 +324,22 @@ define upf_pfcp_policer_show_reply {
   u32 cir_pps;
   u32 cb_ms;
 };
+
+autoreply define upf_pfcp_heartbeats_set {
+  u32 client_index;
+  u32 context;
+  u32 timeout;
+  u32 retries;
+};
+
+define upf_pfcp_heartbeats_show {
+  u32 client_index;
+  u32 context;
+};
+
+define upf_pfcp_heartbeats_show_reply {
+  u32 client_index;
+  u32 context;
+  u32 timeout;
+  u32 retries;
+};
```

Comment:
Although we have other `_show` functions in our API already, I don't really like this convention, as APIs don't "show" anything (don't print anything to the console, unlike CLIs), they just return some info.
In this particular case, I'd name the APIs `upf_pfcp_set_heartbeat_config` and `upf_pfcp_get_heartbeat_config`

Reply:
@ivan4th thanks for noticing about e2e test case
added as a separate commit